modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -21,56 +21,84 @@ definition {
 		}
 	}
 
-	@description = "LPS-144474 - Verify that, when adding a new entry in the Object, the user only is able to submit the entry successfully if it passes for all validations from that Object"
-	@priority = 4
+	@description = "LPS-144474 - Verify it's possible to add valid entries when define output fields for my validation as full validation"
```

Comment:
Verify a valid object entry can be added when the validation is set to full validation